src/main/java/edu/wpi/grip/ui/OperationView.java

```diff
@@ -88,5 +88,31 @@ public void initialize(URL url, ResourceBundle resourceBundle) {
             Step step = new Step(this.eventBus, this.operation);
             this.eventBus.post(new StepAddedEvent(step));
         });
+
+
+        // Ensures that when this element is hidden that it also removes its size calculations
+        this.managedProperty().bind(this.visibleProperty());
+    }
+
+
+    /**
+     * Uses the search string from the palette and allows this element to determine if it should be visible
+     *
+     * @param searchString The search string that should be used to determine visibility
+     */
+    void determineVisibility(String searchString) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/WPIRoboticsProjects/GRIP/file/1273862759/issues/source?bid=2943107&fileBranchId=2943112#l103)